nbodykit/source/catalog/fkp.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, source, BoxSize, Nmesh, dtype, selection, weight, comp_weight
         self.fkp_weight  = fkp_weight
         self.nbar        = nbar
 
-        CatalogMeshSource.__init__(self, source, BoxSize, Nmesh, dtype, weight, selection)
+        CatalogMeshSource.__init__(self, source, BoxSize, Nmesh, dtype, weight, 'Value', selection)
```

Comment:
I wouldn't change FKP at all...it is sort of already handling Weight and Value, which are basically FKPWeight and CompletenessWeight. It follows the nomenclature used in the field for this so I think it's best to leave it as is. We'll document this in the docs.